calc.rb

```diff
@@ -0,0 +1,66 @@
+answer = nil
+puts "Hi there"
+while answer == nil 
+  puts "Please enter one of the following operations, addition, subtraction, \nmultiplication, division, exponent, or modulo:"
+  operation = gets.chomp.downcase
+#gets user input
+  puts "Please enter the first number:"
+  user_num1 = gets.chomp
+  num1 = user_num1.to_f
```

Comment:
It looks like you end up converting this and user_num2 to integers to do some checks in the next if statement and at the very end. Is there a reason why you chose to use floats first and then convert it to integers later? Your code will be more consistent if you stick with one or the other.